opm/simulators/flow/CompositionalContainer.cpp

```diff
@@ -168,16 +168,23 @@ outputRestart(data::Solution& sol,
     this->allocated_ = false;
 }
 
-#define INSTANTIATE_COMP(NUM) \
-    template<class T> using FS##NUM = GenericOilGasFluidSystem<T, NUM>; \
+#define INSTANTIATE_COMP_THREEPHASE(NUM) \
+    template<class T> using FS##NUM = GenericOilGasWaterFluidSystem<T, NUM, true>; \
     template class CompositionalContainer<FS##NUM<double>>;
 
-INSTANTIATE_COMP(0)
+#define INSTANTIATE_COMP_TWOPHASE(NUM) \
+    template<class T> using GFS##NUM = GenericOilGasWaterFluidSystem<T, NUM, false>; \
+    template class CompositionalContainer<GFS##NUM<double>>;
+
+#define INSTANTIATE_COMP(NUM) \
+    INSTANTIATE_COMP_THREEPHASE(NUM) \
+    INSTANTIATE_COMP_TWOPHASE(NUM)
+
+INSTANTIATE_COMP_THREEPHASE(0)  // \Note: to register the parameter ForceDisableFluidInPlaceOutput
 INSTANTIATE_COMP(2)
 INSTANTIATE_COMP(3)
 INSTANTIATE_COMP(4)
 INSTANTIATE_COMP(5)
 INSTANTIATE_COMP(6)
-INSTANTIATE_COMP(7)
```

Comment:
Is this intentional or accidental? If this is intentional, we need to adjust some statement regarding the number of the components that are supported by us.

Reply:
okay, turns out i was just me being sloppy. it's needed and it is in fipcontainer, i somehow missed a line.